reason-parser/src/reason_parser.mly

```diff
@@ -1200,7 +1201,7 @@ _use_file:
 parse_core_type:
     core_type EOF
       {
-        let core_loc = mklocation $startpos($1) $endpos($1) in
+        let core_loc = mklocation $symbolstartpos $endpos($1) in
```

Comment:
It would be interesting to see if it ends up effecting the locations. Does it affects more than *just* optionals? Suppose you have empty sequences (like attributes)?